packages/block-library/src/image/editor.scss

```diff
@@ -21,6 +21,19 @@
 		margin-top: -9px;
 		margin-left: -9px;
 	}
+
+	// These need specificity to override an inherited left/right block margin in the editor.
+	&.wp-block-image.alignleft {
```

Comment:
Can it be generally applied to any block?

Reply:
Yes, probably. There's a generic float rule, as mentioned, that looks like this:

```
.block-editor-block-list__layout .block-editor-block-list__block[data-align="left"] > .is-block-content {
    float: left;
    margin-right: 2em;
}
```

But of course that only applies to blocks that have the "is-block-content" container. I can't recall the heuristic for when such a container.